bin/sv3_obstatus

```diff
@@ -0,0 +1,1537 @@
+        htmlmain.write(
+            "<p style='font-size:1vw; text-align:right'><i>Last updated: {}</p></i>\n".format(
+                js
+            )
+        )
+        htmlmain.write("</html></body>\n")
+        htmlmain.close()
+
+        # ADM make sure all of the relevant directories and plots can be read by a web-browser.
+        cmd = "chmod 644 {}/*".format(outfns["html"])
+        ok = os.system(cmd)
+        cmd = "chmod 775 {}".format(outfns["html"])
+        ok = os.system(cmd)
+
+
+# =====================================
+
+if __name__ == "__main__":
+
+    # AR reading arguments
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--outdir",
+        help="output directory (fiducial location: $DESI_ROOT/survey/observations/SV3/)",
+        type=str,
+        default=None,
+        required=True,
+    )
+    parser.add_argument(
+        "--plot",
+        help="do plots (skymap, observing conditions, r_depth)? (y/n)",
+        type=str,
+        default="y",
+    )
+    parser.add_argument(
+        "--html", help="create html pages, one per tile (y/n)", type=str, default="y",
+    )
+    parser.add_argument(
+        "--numproc",
+        help="number of concurrent processes to use (default=1)",
+        type=int,
+        default=1,
+    )
+    args = parser.parse_args()
+    #
+    if args.outdir[-1] == "/":
+        args.outdir = args.outdir[:-1]
+    # AR printing arguments
+    for kwargs in args._get_kwargs():
+        print(kwargs)
+
+    # AR desitarget version of the sv3 catalogs
+    dtver = "0.57.0"
+
+    # AR rosettes infos
+    # AR https://desi.lbl.gov/trac/wiki/SurveyOps/OnePercent#Rosettefootprints
+    rosette_fields = np.array(
+        [
+            "COSMOS",
+            "GAMA G12",
+            "GAMA G12",
+            "GOODS-North",
+            "Coma cluster",
+            "VVDS-F14",
+            "DEEP2 CFHTLS-D3/W3",
+            "Bootes NDWFS/AGES",
+            "GAMA G15",
+            "GAMA G15",
+            "GAMA G15",
+            "ELAIS N1",
+            "HSC DR2",
+            "HSC DR2",
+            "XDEEP",
+            "Euclid Deep Field",
+        ]
+    )
+    rosette_ras = np.array(
+        [
+            150.100,
+            179.600,
+            183.100,
+            189.900,
+            194.750,
+            210.000,
+            215.500,
+            217.800,
+            216.300,
+            219.800,
+            218.050,
+            242.750,
+            241.050,
+            245.880,
+            252.500,
+            269.730,
+        ]
+    )
+    rosette_decs = np.array(
+        [
+            2.182,
+            0.000,
+            0.000,
+            61.800,
+            28.200,
+            5.000,
+            52.500,
+            34.400,
+            -0.600,
+            -0.600,
+            2.430,
+            54.980,
+            43.450,
+            43.450,
+            34.500,
+            66.020,
+        ]
+    )
```

Comment:
I agree it s not great, but I couldn t find any official file with the rosette properties (that s only in the wiki).
@djschlegel : I don t know if it would make sense to create such a "reference" file, e.g. with the rosette R.A./Dec./field name, and the list of the associated tileids.
anyway, that is really sv3-specific.